packages/xo-web/src/xo-app/logs/backup-ng/log-alert-body.js

```diff
@@ -316,14 +327,21 @@ export default decorate([
       setFilter: (_, filter) => () => ({
         filter,
       }),
-      restartVmJob: (_, { vm }) => async (
+      restartVmJob: (_, { force, vm }) => async (
         _,
         { log: { scheduleId, jobId } }
       ) => {
         await runBackupNgJob({
           id: jobId,
-          vm,
           schedule: scheduleId,
+          settings: force
+            ? {
```

Comment:
What do you think about moving the logic to `runBackupNgJob`  implementation ?

Reply:
Yes I think it's good idea to separate it from UI logic

Reply:
And if we wan't to add more properties linked to the force of the VM restart, to not have to add them  on all functions using `runBackupNgJob`.